pkgs/applications/virtualization/oci-seccomp-bpf-hook/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, buildGoModule
+, fetchFromGitHub
+, go-md2man
+, installShellFiles
+, libseccomp
+, linuxPackages
+, pkg-config
+}:
+
+buildGoModule rec {
+  pname = "oci-seccomp-bpf-hook";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "oci-seccomp-bpf-hook";
+    rev = "v${version}";
+    sha256 = "143x4daixzhhhpli1l14r7dr7dn3q42w8dddr16jzhhwighsirqw";
+  };
+  vendorSha256 = null;
+  doCheck = false;
+
+  outputs = [ "out" "man" ];
+  nativeBuildInputs = [
+    go-md2man
+    installShellFiles
+    pkg-config
+  ];
+  buildInputs = [
+    libseccomp
+    linuxPackages.bcc
+  ];
+
+  buildPhase = ''
+    make
+  '';
+
+  postBuild = ''
+    sed -i "s;HOOK_BIN_DIR;$out/bin;g" oci-seccomp-bpf-hook.json
+  '';
+
+  installPhase = ''
+    install -Dm755 bin/* -t $out/bin
+    install -Dm644 oci-seccomp-bpf-hook.json -t $out
+    installManPage docs/*.[1-9]
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/containers/oci-seccomp-bpf-hook";
+    description = ''
+      OCI hook to trace syscalls and generate a seccomp profile
+    '';
+    license = licenses.asl20;
+    maintainers = with maintainers; [ ] ++ teams.podman.members;
```

Comment:
I don't think this should be added to the team as it is an OCI tool and not directly related to podman/cri-o.

Reply:
Not? I mean I can remove the team if you prefer it, but the hook would be used mostly from Podman and CRI-O from my point of view. :)

Reply:
I'd prefer to remove it for now and we can add it to the team if/when it is added to the wrappers.

Reply:
Sorry, I posted this comment and then deleted it by accident:

I'd prefer to remove it for now and add it if/when we add it to the wrappers.

Reply:
Done :+1: